src/tools/clippy/clippy_lints/src/loops/needless_range_loop.rs

```diff
@@ -273,7 +273,7 @@ impl<'a, 'tcx> VarVisitor<'a, 'tcx> {
                     }
                     return false; // no need to walk further *on the variable*
                 },
-                Res::Def(DefKind::Static(_) | DefKind::Const, ..) => {
+                Res::Def(DefKind::Static{..} | DefKind::Const, ..) => {
```

Comment:
The lack of autoformatting for clippy is evident here, all the clippy changes lack sufficient spacing around the braces.

Reply:
clippy will get formatted after a sync on the clippy repo